supplementary_style_guide/style_guidelines/release-notes.adoc

```diff
@@ -83,7 +83,7 @@ If available, inform users of alternative capabilities and workarounds.
 ==== Deprecation notice
 [subs="+quotes"]
 ----
-In __<product_name> <release>__, __<name_of_capability_or_feature>__ is deprecated and is planned to be removed in the __<deprecation_timeline>__. Red Hat will provide bug fixes and support for this feature during the current release lifecycle, but this feature will no longer receive enhancements and will be removed. As an alternative to __<name_of_capability_or_feature>__, you can use __<alternative_capability_or_feature_if_available>__ instead.
+In __<product_name> <release>__, __<name_of_capability_or_feature>__ is deprecated and is planned to be removed in the __<deprecation_timeline>__. Red{nbsp}Hat will provide bug fixes and support for this feature during the current release lifecycle, but this feature will no longer receive enhancements and will be removed. As an alternative to __<name_of_capability_or_feature>__, you can use __<alternative_capability_or_feature_if_available>__ instead.
```

Comment:
Because this is a code block, the nbsp isn't reformatted as a space. Do we want to leave in the nbsp as an example of it in use, or remove it for clarity? Ditto below.

Reply:
+1

Reply:
Agree wit @spank50 and @sbmetz